src/site/stages/build/process-cms-exports/schemas/raw/node-event.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    uid: { $ref: 'EntityReferenceArray' },
+    changed: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string' },
+          format: { type: 'string' },
+        },
+        required: ['value', 'format'],
+      },
+      maxItems: 1,
+    },
+    path: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          alias: { type: 'string' },
+          langcode: { type: 'string' },
+          pathauto: { type: 'number' },
+        },
+        required: ['alias', 'langcode', 'pathauto'],
+      },
+      maxItems: 1,
+    },
+    metatag: {
+      // Probably should be a common schema
```

Comment:
Not a bad idea, especially since several of the remaining transformers are `node` entities which will have metatags.

I'm ok with that being it's own PR though since this one is pretty hefty like you said.